Podfile

```diff
@@ -143,7 +143,7 @@ abstract_target 'Apps' do
   pod 'NSURL+IDN', '~> 0.4'
 
   # pod 'WordPressAuthenticator', '~> 8.0'
-  pod 'WordPressAuthenticator', git: 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git', commit: 'b51b4b238103f7812db0aa1a265995e1c6d1e355'
+  pod 'WordPressAuthenticator', git: 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git', commit: '215c77f4dad18b67943bc46174050d8c668c597c'
```

Comment:
This resolves the iPad issue. However I am not sure what the pod target here should be. It was targeting a specific commit before as well but this doesn't sounds great. @mokagio perhaps you would have more information about this? Is there a reason why this pod isn't targeting a specific release?

Reply:
> This resolves the iPad issue.

That's great. I noticed [there's not a PR for it yet](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/compare/cobranding-login-prolog-update) but I assume one will be opened soon, right?

![image](https://github.com/wordpress-mobile/WordPress-iOS/assets/1218433/48bf2cfc-211b-48de-bf07-042e0be81cc5)

> However I am not sure what the pod target here should be. It was targeting a specific commit before as well but this doesn't sounds great. 

It's okay to target commits for the pods during development. That is, only `release/` branches should always point to stable pods and other libraries. More info at paNNhX-K4-p2

The one thing to be aware when changing from one commit to another is to make sure that the new commit is after the original one, so that both changes are present. That is, of course, unless the second commit directly aims at fixing an issue in the orginal one, for example by switching for "experimental branch for feature X number 1" to "experimental branch for feature X number 2".

> Is there a reason why this pod isn't targeting a specific release?

The procedure above should answer this question, too. For reference, the PR that changed it is https://github.com/wordpress-mobile/WordPress-iOS/pull/21904.

Hope this helps.